shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/data/remote/model/TokenApiModel.kt

```diff
@@ -0,0 +1,42 @@
+package co.nimblehq.avishek.phong.kmmic.data.remote.model
+
+import co.nimblehq.avishek.phong.kmmic.domain.model.Token
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class TokenApiModel(
+    @SerialName("id")
+    val id: String,
+    @SerialName("type")
+    val type: String,
+    @SerialName("access_token")
+    val accessToken: String,
+    @SerialName("token_type")
+    val tokenType: String,
+    @SerialName("expires_in")
+    val expiresIn: Int,
+    @SerialName("refresh_token")
+    val refreshToken: String,
+    @SerialName("created_at")
+    val createdAt: Int
+) {
+    constructor(token: Token) : this(
+        "-",
+        "-",
+        token.accessToken,
+        token.tokenType,
+        token.expiresIn,
+        token.refreshToken,
+        token.createdAt
+    )
```

Comment:
I believe the intention here is for converting `Token` to `TokenApiModel`.

We usually separate this into another method like `toToken` method you defined below.
```
fun Token.toApiModel() = TokenApiModel(
    id = "",
    type = "",
    accessToken = accessToken,
    tokenType = tokenType,
    expiresIn = expiresIn,
    refreshToken = refreshToken,
    createdAt = createdAt,
)
```

Reply:
Thanks for the advice, let me try that idea

Reply:
@kaungkhantsoe, I improved that in 7d6309e